### ISSUE TITLE
[Bug]: querynode has too many logs

### ISSUE THREAD

**Issue by iytprince2** [milvus-io/milvus#26991]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.2.13
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka):pulsar   
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS):
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

there are too many logs in querynode causing k8s node crashes, the logs like
2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]
[2023/09/11 03:52:55.228 +00:00] [WARN] [flowgraph/node.go:204] ["Invalid operate message input"] ["input length"=0]

### Expected Behavior

there are little logs

### Steps To Reproduce

```markdown
noop
```

### Milvus Log

noop

### Anything else?

noop

**Comment by yanliang567** [comment]
@jiaoew1991 what these logs for, can we improve that?
/assign @jiaoew1991 
/unassign

**Comment by jiaoew1991** [comment]
These logs indicate that there is channel multiplexing, and during recovery, data from other collections needs to be filtered out.
These log contents are indeed useless, we need to limit the frequency.

@congqixia what do you think?

**Comment by xiaofan-luan** [comment]
we should remove it agreed

**Comment by jiaoew1991** [comment]
/assign @smellthemoon 
/unassign 
plz help on this issue

**Comment by hakan458** [comment]
This was a very costly bug for us as we are running Milvus cluster in GKE, and using cloud logging to store logs. This was spamming logs at an extreme rate and cost us over $70,000 just to store these useless logs which we didnt realize at first.

**Comment by xiaofan-luan** [comment]
> This was a very costly bug for us as we are running Milvus cluster in GKE, and using cloud logging to store logs. This was spamming logs at an extreme rate and cost us over $70,000 just to store these useless logs which we didnt realize at first.

sorry for the confusion. Could you turn the log into info level?

**Comment by xiaofan-luan** [comment]
and please let us know what is the most frequent log if possible

**Comment by smellthemoon** [comment]
fixed with #27015

**Comment by yanliang567** [comment]
close

**Comment by dzqoo** [comment]
I got  other log under this frequency logs 
"delete in historical replica " which is alse frequency.
I wonder why I got this log and how to suppress it.

![73c11d19acbcf15d63e7410784c0b91](https://github.com/milvus-io/milvus/assets/8862305/233fafcb-cada-428d-a446-0872d180d518)

My Milvus is 2.2.14.

**Comment by xiaofan-luan** [comment]
/assign @smellthemoon 
can we backport this to 2.2?

**Comment by smellthemoon** [comment]
> /assign @smellthemoon
> can we backport this to 2.2?

it's not a bug-fix, it is necessary to do that in 2.2? it's ok to do this in 2.2.

**Comment by smellthemoon** [comment]
cp:https://github.com/milvus-io/milvus/pull/27046
